ios/Exponent/Versioned/Core/EXVersionManager.mm

```diff
@@ -327,6 +327,17 @@ - (NSArray *)extraModulesForBridge:(id)bridge
   UMModuleRegistry *moduleRegistry = [moduleRegistryAdapter moduleRegistryForParams:params forExperienceId:experienceId withKernelServices:services];
   NSArray<id<RCTBridgeModule>> *expoModules = [moduleRegistryAdapter extraModulesForModuleRegistry:moduleRegistry];
   [extraModules addObjectsFromArray:expoModules];
+  
+  if (!RCTTurboModuleEnabled()) {
+    [extraModules addObject:[self getModuleInstanceFromClass:[self getModuleClassFromName:"DevSettings"]]];
+    id exceptionsManager = [self getModuleInstanceFromClass:RCTExceptionsManagerCls()];
+    if (exceptionsManager) {
+      [extraModules addObject:exceptionsManager];
+    }
+    [extraModules addObject:[self getModuleInstanceFromClass:[self getModuleClassFromName:"DevMenu"]]];
+    [extraModules addObject:[self getModuleInstanceFromClass:[self getModuleClassFromName:"RedBox"]]];
+    [extraModules addObject:[self getModuleInstanceFromClass:RCTAsyncLocalStorageCls()]];
```

Comment:
```suggestion
	NSArray *modulesClasses = @[
	  [self getModuleClassFromName:"DevSettings"],
	  RCTExceptionsManagerCls(),
	  [self getModuleClassFromName:"DevMenu"],
	  [self getModuleClassFromName:"RedBox"],
	  RCTAsyncLocalStorageCls(),
	];
	for (id moduleClass in modulesClasses) {
      id module = [self getModuleInstanceFromClass:moduleClass]
	  if (module) {
        [extraModules addObject:module];
      }
	}
```
I wish we could go like this:
1. Create initial array containing modules classes
2. Map this classes into sth
3. Filter out nil values
4. Use [NSMutableArray#addObjects](https://developer.apple.com/documentation/foundation/nsmutablearray/1408963-addobjects)

Reply:
We talked about this on Slack and decided it's not required for the PR to be merged, so dismissing. Reasons for the current way of implementation:
- similar to what previously was
- extremely simple
- git-safe too